resources/README.md

```diff
@@ -0,0 +1,8 @@
+# Resources
+
+Here are a list of resources that are useful to our team.
+
+## Pictures
+
+- [Pexels](https://www.pexels.com/): royalty free images & videos
```

Comment:
I wonder why the linting check returned this link https://www.pexels.com/ as a dead link

Reply:
It's because it use curl, and they use Cloudflare for DDoS protection. I added this link to the ignore file for the link checker action.